ui/component/settingsStripeCard/view.jsx

```diff
@@ -230,23 +245,32 @@ const SettingsStripeCard = (props: Props) => {
           type="input"
           label={__('Card details')}
           inputElem={
-            <CardElement
-              className="stripe-card__form-input"
-              onReady={(element) => {
-                cardElement.current = element;
-              }}
-              onChange={(event) => setFormError(event.error?.message)}
-            />
+            <CardElement className="stripe-card__form-input" onChange={(event) => setFormError(event.error?.message)} />
           }
         />
 
-        <Submit
-          className="button--card-link"
-          disabled={isLoading || formError || !cardNameValue}
-          label={isLoading ? <div className="stripe__spinner" /> : __('Add Card')}
-        />
+        {stripeError ? (
+          <Button
+            className="button--card-link"
+            label={promisePending ? <div className="stripe__spinner" /> : __('Retry')}
+            onClick={reloadForm}
+          />
+        ) : (
+          <Submit
+            className="button--card-link"
+            disabled={isLoading || formError || !cardNameValue}
+            label={isLoading ? <div className="stripe__spinner" /> : __('Add Card')}
+          />
+        )}
 
-        {formError && <ErrorText>{formError}</ErrorText>}
+        {formError && (
+          <span className="error__text error__text--stripe-card">
+            <p>{formError}</p>
+            {stripeError ? (
+              <p>{__('You may have blockers turned on, try turning them off and reloading.')}</p>
+            ) : undefined}
```

Comment:
1. Block stripe.js
2. Message appears
3. Type something in the name field -- the error message never appears again.  Can live with it though, since the user already knows the situation.
4. Press Retry -- error message did not reappear, although this could just be due to the first issue above.